### PR TITLE
feat: show properties cart

### PR DESCRIPTION
**Pull request by FabianGerke**
## Why:

Closes: [AB#90293](https://dev.azure.com/plentymarkets/0af57df9-8662-4901-bb97-8e88b07ff0c9/_workitems/edit/90293)

## Checklist before requesting a review

- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I've updated `docs/changelog/changelog_en.md` and `docs/changelog/changelog_de.md`. If I'm a non-German speaker, I've still updated the file with the English version as a placeholder.

### CODE REVIEW

File: apps/web/components/CartOrderProperty/CartOrderProperty.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template>
+  <div v-if="property" class="list-item list-disc ml-4">
+    <span class="font-bold">
+      {{ basketItemOrderParam.name }}
+      <span v-if="productPropertyGetters.getOrderPropertyLabel(property).surchargeType">
+        ({{ $t('orderProperties.vat.' + productPropertyGetters.getOrderPropertyLabel(property).surchargeType) }}
+        {{ $n(productPropertyGetters.getOrderPropertySurcharge(property), 'currency') }})</span
+      >
+    </span>
+    <span v-if="productPropertyGetters.getOrderPropertyValueType(property) !== 'empty'">: {{ value }}</span>
```
Comment (maxiroellplenty) [inline apps/web/components/CartOrderProperty/CartOrderProperty.vue]: !isOrderPropertyCheckbox(property)

File: apps/web/components/CartPageContent/CartPageContent.vue [hunk 2]
```diff
@@ -8,19 +8,21 @@
...
-          :tag="NuxtLink"
-          :to="localePath(paths.checkout)"
-          size="lg"
-          class="w-full mb-4 md:mb-0"
-        >
-          {{ $t('goToCheckout') }}
-        </SfButton>
-        <PayPalExpressButton class="mt-4" type="CartPreview" />
-      </OrderSummary>
+      <client-only>
```
Comment (maxiroellplenty) [inline apps/web/components/CartPageContent/CartPageContent.vue]: why?

File: apps/web/components/OrderSummary/OrderSummary.vue [hunk 3]
```diff
@@ -22,14 +42,34 @@
...
+          v-for="property in orderPropertiesWithoutVat"
+          :key="property.propertyId"
+        >
+          <div class="flex flex-col gap-2 grow pr-2">
+            <p>{{ property.name }}</p>
+          </div>
+          <div class="flex flex-col gap-2 text-right">
+            <p>{{ n(property.price, 'currency') }}</p>
+          </div>
+        </div>
```
Comment (maxiroellplenty) [inline apps/web/components/OrderSummary/OrderSummary.vue]: could use getters

**Review by maxiroellplenty — CHANGES_REQUESTED** [review]

File: apps/web/components/CartOrderProperty/CartOrderProperty.vue (hunk 1 above)
Comment (FabianGerke) [reply apps/web/components/CartOrderProperty/CartOrderProperty.vue]: Smart

**Review by maxiroellplenty — APPROVED** [review]

**Pull request merged**